Core/Newsletter2Go/Controllers/Backend/Newsletter2go.php

```diff
@@ -17,11 +17,8 @@ class Shopware_Controllers_Backend_Newsletter2go extends Shopware_Controllers_Ba
      *
      * @throws \Exception
      */
-    public function __construct(
-        Enlight_Controller_Request_Request $request = null,
-        Enlight_Controller_Response_Response $response = null
-    ) {
-        parent::__construct($request, $response);
+    public function __construct() {
```

Comment:
actually we already have a model Newsletter2Go that extends `ModelEntity` which means that it makes sense to extend `Shopware_Controllers_Backend_Application` instead to use CRUD functionalities.
I just feel that this change should be done in a different ticket since a big refactoring will be needed and a lot in the plugin can break. wdyt?

Reply:
Sure - maybe create a ticket and rework this later on, okay?!